tests/UnifiedSpecTests/UnifiedSpecTest.php

```diff
@@ -61,6 +61,56 @@ class UnifiedSpecTest extends FunctionalTestCase
+        'crud/insertMany-comment: insertMany with comment - pre 4.4' => 'Not yet implemented (PHPC-2049, PHPLIB-749)',
+        'crud/insertOne-comment: insertOne with string comment' => 'Not yet implemented (PHPC-2049, PHPLIB-749)',
+        'crud/insertOne-comment: insertOne with document comment' => 'Not yet implemented (PHPC-2049, PHPLIB-749)',
+        'crud/insertOne-comment: insertOne with comment - pre 4.4' => 'Not yet implemented (PHPC-2049, PHPLIB-749)',
+        'crud/replaceOne-comment: ReplaceOne with string comment' => 'Not yet implemented (PHPC-2049, PHPLIB-749)',
+        'crud/replaceOne-comment: ReplaceOne with document comment' => 'Not yet implemented (PHPC-2049, PHPLIB-749)',
+        'crud/replaceOne-comment: ReplaceOne with comment - pre 4.4' => 'Not yet implemented (PHPC-2049, PHPLIB-749)',
+        'crud/updateMany-comment: UpdateMany with string comment' => 'Not yet implemented (PHPC-2049, PHPLIB-749)',
+        'crud/updateMany-comment: UpdateMany with document comment' => 'Not yet implemented (PHPC-2049, PHPLIB-749)',
+        'crud/updateMany-comment: UpdateMany with comment - pre 4.4' => 'Not yet implemented (PHPC-2049, PHPLIB-749)',
+        'crud/updateOne-comment: UpdateOne with string comment' => 'Not yet implemented (PHPC-2049, PHPLIB-749)',
+        'crud/updateOne-comment: UpdateOne with document comment' => 'Not yet implemented (PHPC-2049, PHPLIB-749)',
+        'crud/updateOne-comment: UpdateOne with comment - pre 4.4' => 'Not yet implemented (PHPC-2049, PHPLIB-749)',
+        'crud/aggregate: aggregate with comment sets comment on getMore' => 'Not yet implemented (PHPC-2049, PHPLIB-749)',
+        'crud/aggregate: aggregate with a document comment' => 'Not yet implemented (PHPC-2049, PHPLIB-749)',
+        'crud/aggregate: aggregate with a document comment - pre 4.4' => 'Not yet implemented (PHPC-2049, PHPLIB-749)',
+        // Some crud operations tests for "let" option don't specify "multi" and "upsert" fields in the expected events
+        'crud/updateOne-let: UpdateOne with let option' => 'Not yet implemented (DRIVERS-2270)',
+        'crud/updateOne-let: UpdateOne with let option unsupported (server-side error)' => 'Not yet implemented (DRIVERS-2270)',
+        'crud/updateMany-let: updateMany with let option' => 'Not yet implemented (DRIVERS-2270)',
+        'crud/updateMany-let: updateMany with let option unsupported (server-side error)' => 'Not yet implemented (DRIVERS-2270)',
+        'crud/replaceOne-let: ReplaceOne with let option' => 'Not yet implemented (DRIVERS-2270)',
+        'crud/replaceOne-let: ReplaceOne with let option unsupported (server-side error)' => 'Not yet implemented (DRIVERS-2270)',
+        'crud/BulkWrite replaceOne-let: BulkWrite replaceOne with let option' => 'Not yet implemented (DRIVERS-2270)',
+        'crud/BulkWrite replaceOne-let: BulkWrite replaceOne with let option unsupported (server-side error)' => 'Not yet implemented (DRIVERS-2270)',
```

Comment:
The "comment" skips can stay, as they'll end up in the final PR.

But let's remove the skips for "let" since it'd be helpful to see the CI failures until the spec tests are synced. And you may be able to do so now since I published https://github.com/mongodb/specifications/pull/1174.